nuget.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nventive" value="https://pkgs.dev.azure.com/nventive/_packaging/nventive/nuget/v3/index.json" />
+    <add key="NuGet official package source" value="https://api.nuget.org/v3/index.json" />
```

Comment:
use "nuget.org" as key?

Reply:
the key is the label name for this. It can be anything. Both "NuGet official package source"  and nuget.org look fine to me.

Reply:
Updated